packages/documentation-site/patternfly-docs/content/developer-resources/dark-theme-handbook.md

```diff
@@ -0,0 +1,65 @@
+---
+id: Dark theme handbook
+title: PatternFly dark theme handbook
+section: developer-resources
+---
+
+## Enabling dark theme
+
+Dark theme styles ship with PatternFly CSS by default. To enable dark theme, add the class  `pf-v5-theme-dark`  to the `<html>` tag of your application. This class can be added dynamically to toggle dark theme on and off in your application.
+
+Dark theme can also be applied based on the browser’s `prefers-color-scheme` media query. The recommended way to apply dark theme based on this media query is to use Javascript(JS) to detect the media query and apply the `pf-theme-dark` class via the JS code. This allows a product to simply manipulate a single class to turn dark theme on or off based on user preference. Adam Argyle wrote [a great article](https://web.dev/building-a-theme-switch-component/#javascript) on using JS to manage dark theme preference, using the `prefers-color-scheme` media query alongside a manual button to toggle dark theme on or off. 
+
+## What’s changed?
+The following are some notable changes and differences between light and dark themes. 
+
+| Color palette changes | Variable(s) affected | New or updated color values|
+| ---------------| -------------------- | -------------------- |
+|Black color palette updated and a new lighter gray was added| --pf-v5-global--palette--black-50  | #e0e0e0 |
+|                           | --pf-v5-global--palette--black-100 | #c6c7c8 |
+|                           | --pf-v5-global--palette--black-200 | #aaabac |
+|                           | --pf-v5-global--palette--black-300 | #868789 |
+|                           | --pf-v5-global--palette--black-400 | #57585a |
+|                           | --pf-v5-global--palette--black-500 | #444548 |
+|                           | --pf-v5-global--palette--black-600 | #36373a |
+|                           | --pf-v5-global--palette--black-700 | #26292d |
+|                           | --pf-v5-global--palette--black-800 | #1b1d21 |
+|                           | --pf-v5-global--palette--black-850 | #212427 |
+|                           | --pf-v5-global--palette--black-900 | #0f1214 |
+|                           | --pf-v5-global--palette--black-1000| #030303 |
+|New red colors added       | --pf-v5-global--palette--red-9999  | #fe5142 |
+|                           | --pf-v5-global--palette--red-8888  | #ff7468 |
+|New primary blue added     | --pf-v5-global--palette--blue-300 | #1fa7f8 |
+
+
+
+**Global variables added**
+
+`--pf-v5-global--BackgroundColor--300` added as background color for elements that appear on top of others. Examples include popovers and dropdown menus.
+
+`--pf-v5-global--BackgroundColor--400` added as background color for hover/interaction state of items placed on top of other dark background. Examples include hover/focus backgrounds for items in a dropdown menu or popover and control elements like text inputs, textareas, menu toggles.
```

Comment:
I'm not sure what a good description is for this color, but looking at the code, the majority of uses are

control elements
* the button "control" variation
* menu toggle (includes select, dropdown, etc)
* anything that presents as a "form control" - like text inputs and text areas

hover/interaction state of an item places on top of `--pf-v5-global--BackgroundColor--300` - the floating background listed above.

wdyt @mceledonia 

```suggestion
`--pf-v5-global--BackgroundColor--400` added as background color for hover/interaction state of items placed on top of other dark background. Examples include hover/focus backgrounds for items in a dropdown menu or popover and control elements like text inputs, textareas, menu toggles.
```